easybuild/easyconfigs/i/ifort/ifort-2018.0.128-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,40 @@
+# This is an easyconfig file for EasyBuild, see http://easybuilders.github.io/easybuild
+
+name = 'ifort'
+version = '2018.0.128'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "Intel Fortran compiler"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_fortran.tgz']
+
+checksums = [
+    # parallel_studio_xe_2018_composer_edition_for_fortran.tgz
+    '597718bf752a52e043675102c6c03971be5dd3400a2e849bc295094395beef89',
+    '64793f16491eb35b1c008eaacc348bff283a53f78ba766e5ece7878a38ae6064',   # ifort_2017_no_mpi_mic_dependency.patch
+]
+
+# remove dependency on intel-mpi-rt-mic
+patches = ['ifort_2017.09_no_mpi_mic_dependency.patch']
```

Comment:
@reedts It probably makes sense to rename this to `ifort_2018_no_mpi_mic_dependency.patch`?

Do you mind adding you as an author in the patch (incl. a mention of Bart Oldeman, since he's the author of the original patch)? Which name/affiliation should we use?